src/Components/Endpoints/src/PublicAPI.Unshipped.txt

```diff
@@ -48,10 +16,6 @@ Microsoft.AspNetCore.Components.Endpoints.IComponentPrerenderer.PrerenderCompone
 Microsoft.AspNetCore.Components.Endpoints.IComponentPrerenderer.PrerenderPersistedStateAsync(Microsoft.AspNetCore.Http.HttpContext! httpContext, Microsoft.AspNetCore.Components.PersistedStateSerializationMode serializationMode) -> System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Html.IHtmlContent!>
 Microsoft.AspNetCore.Components.Endpoints.IRazorComponentsBuilder
```

Comment:
Can we make the `IRazorComponentsBuilder` namespace change in this PR since that's the only thing that looks to be remaining from #49754?

```suggestion
Microsoft.Extensions.DependencyInjection.IRazorComponentsBuilder
```